src/index.js

```diff
@@ -47,79 +46,39 @@ export default class FormPlugin extends Plugin {
         // no control key pressed
         if (!event.metaKey) {
```

Comment:
I have always checked for more special `event` properties, like this:
```
if (e.metaKey || e.ctrlKey || e.altKey || e.shiftKey) return false;
```
Actually just had a look at swup's source code, it's doing the same inside `linkClickHandler`: https://github.com/swup/swup/blob/master/src/index.js#L197

Sorry, unrelated to the PR but oh well, while we are on it ;)

Reply:
See [SubmitEvent](https://developer.mozilla.org/en-US/docs/Web/API/SubmitEvent) on MDN

Reply:
Oh nice catch! First thought would be a `keyDown` listener that detects this so that we don't break default browser behavior, right? As it is now, the `openFormSubmitInNewTab` event would never be fired, at least in chrome

Reply:
I'd suggest undoing the `metaKey` changes to leave everything as-is for now and opening a new issue so  we can do some more research.